tests/integration/test_metrics.py

```diff
@@ -36,14 +36,18 @@ def _session_payload(timestamp: datetime, started: datetime):
     }
 
 
-def metrics_by_name(metrics_consumer, count, timeout=None):
+def metrics_by_name(metrics_consumer, count=None, timeout=None):
     metrics = {"headers": {}}
 
-    for _ in range(count):
-        metric, metric_headers = metrics_consumer.get_metric(timeout)
+    attempts = 99 if count is None else count
```

Comment:
```suggestion
    attempts = count or 99
```

Reply:
It's theoretically valid to call the function with `count=0` to assert that the topic is empty, so I prefer handling `None` explicitly.